spec/meilisearch/index/search/nested_fields_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+RSpec.describe 'MeiliSearch::Index - nested fields search' do
+  include_context 'search books with nested fields'
+
+  it 'searches without params' do
+    response = index.search('an awesome')
+
+    expect(response['hits'].count).to eq(1)
+    expect(response.dig('hits', 0, 'info', 'comment')).to eq('An awesome book')
+    expect(response.dig('hits', 0, 'info', 'reviewNb')).to eq(900)
+  end
+
+  it 'searches with index with searchableAttributes setting' do
```

Comment:
why do you add `with index` here in the name but not in the previous test?

Reply:
I think in this case I wanted to write: `searches within index with searchableAttributes setting`

Reply:
WDYT?

Reply:
Yes I think it is the most clear